src/processing/style.ts

```diff
@@ -109,12 +111,7 @@ class StyleProcessor {
     }
 
     private _replaceUrlAttributes (css: string): string {
-        return css.replace(ATTRIBUTE_SELECTOR_RE, (match, prefix, prev, possibleTag, attribute, postfix) => {
-            const tagName = prev === '.' || prev === '#' ? '' : possibleTag;
-
-            if (!tagName || !URL_ATTR_TAGS[attribute] || URL_ATTR_TAGS[attribute].indexOf(tagName) === -1)
-                return match;
-
+        return css.replace(ATTRIBUTE_SELECTOR_RE, (_, prefix, attribute, postfix) => {
```

Comment:
Can we make a performance server tests [here](https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/server/performance-test.js)?